### PR TITLE
Deployment including VerneMQ and new components

### PR DESCRIPTION
**Pull request by aluisribeiro**
* **Please check if the PR fulfills these requirements**
  * [ ] Tests for the changes have been added (for bug fixes / features)
  * [ ] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
Include VerneMQ and new components

* **What is the current behavior?** (You can also link to an open issue here)
Lacks verneMQ and new components

* **What is the new behavior (if this is a feature change)?**
Include VerneMQ and new components

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
Yes, this PR is connected to dojot/dojot#1495

* **Other information**:

### CODE REVIEW

File: roles/minio/defaults/main.yaml [hunk 1]
```diff
@@ -8,6 +8,4 @@ dojot_minio_port: 9000
 
 dojot_minio_cluster_size: 1
 
-dojot_minio_persistent_volumes: false
-
-dojot_minio_volume_size: 4Gi
+dojot_minio_volume_size: 2Gi
```
Comment (eduardogmisiuk) [inline roles/minio/defaults/main.yaml:11]: Shouldn't we have a dojot_minio_version variable too?

**Comment by mprevide** [comment]
You removed nodeports... Which ports are used now?
It may be nice to update this in the documentation.
I don't know it well, sorry if my question doesn't make sense.

File: roles/ejbca-node/templates/ejbca_deployment.yaml.j2 [hunk 2]
```diff
@@ -5,35 +5,32 @@ metadata:
...
+  template:
     metadata:
       labels:
         app: ejbca-wrapper
     spec:
       hostname: localhost
       containers:
       - name: ejbca-wrapper
-        image: dojot/ejbca-wrapper:{{ dojot_version  }}  ## change to dojot_ejbca_version
+        image: dojot/ejbca-wrapper:{{ dojot_version  }}
```
[the remark this answers is not in the document]
Comment (andersonluisribeiro) [reply roles/ejbca-node/templates/ejbca_deployment.yaml.j2]: Ok, that has been changed.

File: roles/minio/defaults/main.yaml (hunk 1 above)
Comment (andersonluisribeiro) [reply roles/minio/defaults/main.yaml:13]: Ok, that has been changed.

**Comment by aluisribeiro** [comment]
Hi @mprevide , NotePorts are being generated automatically so as not to cause port conflicts in the customer's cluster. The user's cluster may already be using these ports.

**Pull request merged**